src/components/CreateNote/CreateNoteContainer.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  UseQueryResult,
+  useMutation,
+  useQuery,
+  useQueryClient,
+} from '@tanstack/react-query';
+import { v4 as uuid } from 'uuid';
+
+import { fetchNotesOfUser, notesActions } from 'api';
+import { INote } from 'types';
+import { useFieldValue, useToggle } from 'hooks';
+
+import CreateNote from './CreateNote';
+import { setNewId } from 'utils';
+import { useAppSelector } from 'store/hooks';
+import { getUser } from 'store/selectors';
+
+const CreateNoteContainer = () => {
+  const queryClient = useQueryClient();
+  const { user } = useAppSelector(getUser);
+  const { data: notes }: UseQueryResult<INote[], Error> = useQuery<
+    INote[],
+    Error,
+    INote[]
+  >(['notes'], () =>
+    user ? fetchNotesOfUser(user.userId) : fetchNotesOfUser(''),
+  );
+  const [isOpen, setOpen] = useToggle(false);
+
+  const { value: title, onChange: onChangeTitle } = useFieldValue();
+  const { value: description, onChange: onChangeDescription } = useFieldValue();
+
+  const { mutate: addNote } = useMutation({
+    mutationFn: notesActions.addNote,
+    onSuccess: () => {
+      queryClient.invalidateQueries(['notes']);
```

Comment:
please add a constant QUERY_KEYS (object) which should contain all query keys and reuse it here like QUERY_KEYS.NOTES